### PR TITLE
Telemetry sending

### PR DESCRIPTION
**Pull request by popovaan**
# What does this PR do?

Adding of Telemetry sending.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/intel/utils/import_utils.py [hunk 1]
```diff
@@ -74,6 +74,21 @@
...
+                          backend='ga4',
+                          enable_opt_in_dialog=False,
+                          disable_in_ci=True)
+
+        # temporal code to check that "CI" variable is set in CI pipelines,
+        # needs to be removed before merge
+        import os
+        assert "CI" in os.environ and os.environ["CI"].lower() == "true"
+
+        tm.send_event("optimum_intel", "openvino_imported", "OpenVino {}, Optimum {}.".format(get_version(), importlib_metadata.version("optimum")))
```
Comment (echarlaix) [inline optimum/intel/utils/import_utils.py:91]: Not sure this is something we want to add in optimum

Comment (popovaan) [reply optimum/intel/utils/import_utils.py]: There was a request to add Telemetry from Gorbachev Yury so we could track the number of users in optimum-intel.

**Review by AlexKoff88 — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

File: optimum/intel/utils/import_utils.py [hunk 2]
```diff
@@ -73,8 +73,24 @@
 _openvino_version = "N/A"
 if _openvino_available:
     try:
+        import openvino_telemetry as tm
```
Comment (eaidova) [inline optimum/intel/utils/import_utils.py:76]: should be here additional try-except for import telemetry only?

if user has openvino but not install openvino-telemetry it now can fail reporting that openvino is not available

File: optimum/intel/utils/import_utils.py (hunk 1 above)
Comment (AlexKoff88) [reply optimum/intel/utils/import_utils.py]: @echarlaix, any concerns about this request? We track OpenVINO usage on a regular basis and we would like to know how much inflow comes from HF ecosystem.

Comment (AlexKoff88) [reply optimum/intel/utils/import_utils.py]: @echarlaix, can you please take a look again? As @popovaan mentioned, openvino-telemetry is installed always when openvino is installed in the environment. We are just enabling it here to track optimum usage. CI jobs are excluded.

**Pull request closed without merging**